tests/test_modelgen.py

```diff
@@ -123,7 +115,8 @@ def test_CNN_hyperparameters_nrlayers(self):
             if key in custom_settings:
                 custom_settings[key] = value
 
-        hyperparams = modelgen.generate_CNN_hyperparameter_set(custom_settings)
+        model_type = Model_CNN(None, None, **custom_settings)
+        hyperparams = model_type.generate_hyperparameters()
```

Comment:
Note that using "None" for inputs and outputs could cause issues in the future, instead of using dummy ones